ucsmsdk/utils/ucstechsupport.py

```diff
@@ -97,6 +97,13 @@ def _set_ts_options_chassis(ts_cmd_opt, kwargs):
             ts_cmd_opt.cimc_adapter_id = TechSupOptsConsts.CIMC_ADAPTER_ID_ALL
     elif _is_valid_arg("iom_id", kwargs):
         ts_cmd_opt.chassis_iom_id = str(kwargs["iom_id"])
+    elif _is_valid_arg("cartridge_id", kwargs):
+        ts_cmd_opt.chassis_cartridge_id = str(kwargs["cartridge_id"])
+        if ts_cmd_opt.chassis_cartridge_id != "all":
+            if _is_valid_arg("cartridge_cimc_id", kwargs):
+                ts_cmd_opt.cartridge_cimc_id = str(kwargs["cartridge_cimc_id"])
+            else:
+                raise UcsValidationException('cartridge_cimc_id should be specified')
```

Comment:
@waghswapnil isn't there a default value for `cartridge_cimc_id` - probably `all`. Wondering if raising an exception here is all we can do?